### PR TITLE
Bau updating post award values

### PR DESCRIPTION
**Pull request by aaronwilliamsv1**
### Change description
Updating authenticator with new secret values and updating the values for post-award hosts

- [ N/A ] Unit tests and other appropriate tests added or updated
- [ N/A ] README and other documentation has been updated / added (if needed)
- [ X ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### CODE REVIEW

**Comment by cyrusdobbs** [comment]
How will this impact the Find tool hosted on the Dev environment at `https://find-monitoring-data.test.gids.dev/`

File: copilot/fsd-authenticator/manifest.yml [hunk 1]
```diff
@@ -62,8 +62,8 @@ variables:
     from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-RedisInstanceURI
 
 secrets:
-  RSA256_PUBLIC_KEY_BASE64: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/RSA256_PUBLIC_KEY_BASE64
-  RSA256_PRIVATE_KEY_BASE64: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/RSA256_PRIVATE_KEY_BASE64
+  RSA256_PUBLIC_KEY_BASE64: /copilot/${COPILOT_APPLICATION_NAME}/${COPILOT_ENVIRONMENT_NAME}/secrets/authenticator_RSA256_PUBLIC_KEY_BASE64
```
Comment (gidsg) [inline copilot/fsd-authenticator/manifest.yml:65]: Shouldn't need to prefix this with `authenticator_` value is shared between apps.

Comment (gidsg) [reply copilot/fsd-authenticator/manifest.yml:65]: Value is already correct and shared, can we revert this change?

File: copilot/fsd-authenticator/manifest.yml [hunk 2]
```diff
@@ -49,8 +49,8 @@ variables:
   FUND_STORE_API_HOST: "http://fsd-fund-store:8080"
   ACCOUNT_STORE_API_HOST: "http://fsd-account-store:8080"
   AUTHENTICATOR_HOST: "https://authenticator.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk"
-  POST_AWARD_FRONTEND_HOST: "https://find-monitoring-data.${data.cloudfoundry_domain.workspace.name}"
-  POST_AWARD_SUBMIT_HOST: "https://submit-monitoring-data.${data.cloudfoundry_domain.workspace.name}"
+  POST_AWARD_FRONTEND_HOST: "https://find-monitoring-data.${COPILOT_ENVIRONMENT_NAME}.access-funding.test.levellingup.gov.uk"
```
Comment (gidsg) [inline copilot/fsd-authenticator/manifest.yml:52]: Can we leave POST_AWARD_FRONTEND_HOST as-is for now until we've fully switched that app over to the new domain?

File: copilot/fsd-authenticator/manifest.yml (hunk 1 above)
Comment (aaronwilliamsv1) [reply copilot/fsd-authenticator/manifest.yml:65]: the RSA256_PUBLIC_KEY_BASE64/RSA256_PRIVATE_KEY_BASE64 are not the same between the _services_ of the application.

Parameter store:

image

As the 'secrets' are stored per _application_ (in this case, preaward), if you have need for multiples of the same named secret (in this case, the public key) then the parameter names need to be unique, or you'd override them

Comment (aaronwilliamsv1) [reply copilot/fsd-authenticator/manifest.yml:65]: After investigation - gonna revert them all to use the same key, for the sake of convenience. At the moment, there is no need to be this specific, and if we need to change how this works, then that should be a seperate block of work. It doesn't change much how this deploys, and saves us some store slots

**Review by gidsg — APPROVED** [review]
👍

**Pull request merged**